utils/grafana-container.bash

```diff
@@ -0,0 +1,12 @@
+set -ex
+docker rm -f grafana || true
+docker run -p 3000:3000 \
+        -e GF_ANALYTICS_REPORTING_ENABLED=false \
+        -e GF_ANALYTICS_CHECK_FOR_UPDATES=false \
+        -e GF_SECURITY_ADMIN_USER=admin \
+        -e GF_SECURITY_ADMIN_PASSWORD=password \
+        -e GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS=tribe-29-checkmk-datasource \
+        -v -v "$(pwd)"/data:/var/lib/grafana \
```

Comment:
is this working? i think docker will complain about the first `-v` without argument? and the plugin is not installed?

my current setup uses `-v "$(pwd)"/dist:/var/lib/grafana/plugins/grafana-checkmk-datasource \` so you lose all dashboards and datasources, but i'm okay with that.

we should add a comment where the script should be executed, because it only works if executed in the root of the project.